financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.extensions/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/extensions/authorize/impl/ExternalAPIConsentRetrievalStep.java

```diff
@@ -82,10 +82,17 @@ public void execute(ConsentData consentData, JSONObject jsonObject) throws Conse
             }
 
             ExternalAPIConsentResourceRequestDTO externalAPIConsentResource = null;
+            String consentFileContent = "";
             if (consentId != null) {
                 DetailedConsentResource detailedConsentResource = consentCoreService.getDetailedConsent(consentId);
                 if (detailedConsentResource != null) {
                     externalAPIConsentResource = new ExternalAPIConsentResourceRequestDTO(detailedConsentResource);
+                    try {
+                        consentFileContent = consentCoreService.getConsentFile(consentId).getConsentFile();
+                        externalAPIConsentResource.setFileContent(consentFileContent);
+                    } catch (ConsentManagementException e) {
+                        log.debug("No consent file found for the given consent Id.");
```

Comment:
why we have put a debug log here?

Reply:
This is not an actual error. The default behavior of the consent core service is to throw the exception when a file is not found.

Reply:
shall we add the consent id to the debug log